### PR TITLE
GH-750 alias examples for NuGet, NUnit, WiX

### PR DESCRIPTION
**Pull request by UnbelievablyRitchie**
Another couple alias examples to move #750 along.

Cheers.

### CODE REVIEW

File: src/Cake.Common/Tools/NuGet/NuGetAliases.cs [hunk 1]
```diff
@@ -806,6 +819,14 @@ public static void NuGetSetApiKey(this ICakeContext context, string apiKey, stri
         /// <summary>
         /// Set the proxy settings to be used while connecting to your NuGet feed, including settings.
         /// </summary>
+        /// <example>
+        /// <code>
+        /// var setting = new NuGetSetApiKeySettings {
```
Comment (gep13) [inline src/Cake.Common/Tools/NuGet/NuGetAliases.cs]: @UnbelievablyRitchie I think this should be `NuGetSetProxySettings`

**Comment by gep13** [comment]
@UnbelievablyRitchie this looks good to me.  Just one small comment.  If you can fix that up, and rebase, then I can get this pulled in.

**Comment by gep13** [comment]
@UnbelievablyRitchie thanks for fixing that? Are you ok to squash the two commits into a single commit?  You can find instructions here: https://github.com/cake-build/cake/blob/develop/CONTRIBUTING.md#respond-to-feedback-on-pull-request

**Comment by gep13** [comment]
Thank you for fixing up the PR! :smile_cat:

**Comment by UnbelievablyRitchie** [comment]
No probs :)

**Pull request merged**